docs/sql-ref-syntax-aux-show-create-table.md

```diff
@@ -19,4 +19,39 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`SHOW CREATE TABLE` returns the `CREATE TABLE` statement that creates a given table. `SHOW CREATE TABLE` on a non-existent table throws Exception.
```

Comment:
@huaxingao Should we show the "CREATE TABLE"  as a link here ? Also how about : 
`SHOW CREATE TABLE` returns the [CREATE TABLE LINK] that was used create a given table ?